### DISCUSSION TITLE
Crash file locations on Windows

### DISCUSSION THREAD

**Discussion by steveP65** [rgriebl/brickstore#919] [General]
Had my Windows HDD crash, replaced with SSD drive and connected old crashed drive via USB, but I can not find the .bsx files that I have saved. Would appreciate some advice. Thanks

**Comment by rgriebl** [comment]
If you are talking about the autosave files that BrickStore is doing every minute: these are in the system's **temp** directory. For Windows this would be: `C:/Users/<USER>/AppData/Local/Temp` the files are named `brickstore_<unique id>.autosave`.
They are in a different format than normal BSX files though, so you can't open them directly. You can quit BrickStore, copy the autosave file to the same location on the new drive and restart BrickStore: it should then tell you it found autosaves and restore them.

**Reply by steveP65** [reply]
Thanks for the reply, yes I see how that autosave works, when you do save it disappears. While searching on the HDD that crashed I have been unable to find it. Brickstore was open at the time when I received the Blue Screen, it was probably only 5 seconds or less after I had added parts and never had a chance to save the file I was working on, so I should have been expecting to find that autosave file? 
I couldn't find it so I have had to hand to a PC Technician to see if he can find it. Hopefully he is running better software than me.
